charset_normalizer/normalizer.py

```diff
@@ -457,7 +461,40 @@ def from_bytes(sequences, steps=10, chunk_size=512, threshold=0.20, cp_isolation
                 int(maximum_length / steps)
             )
 
-            measures = [ProbeChaos(str(sequences[i:i + chunk_size], encoding=p, errors='ignore'), giveup_threshold=threshold, bonus_bom_sig=bom_available, bonus_multi_byte=is_multi_byte_enc) for i in r_]
+            measures = []
+
+            too_much_chaos = False
+
+            for i in r_:
+                measures.append(
+                    ProbeChaos(
+                        str(sequences[i:i + chunk_size], encoding=p, errors='ignore'),
+                        giveup_threshold=threshold,
+                        bonus_bom_sig=bom_available,
+                        bonus_multi_byte=is_multi_byte_enc
+                    )
+                )
+
+                logger.debug(
+                    '{encoding}/measure[{i}] = {chaos_measure} %',
+                    encoding=p,
+                    i=i,
+                    chaos_measure=round(measures[-1].ratio*100, ndigits=3)
+                )
+
+                if measures[-1].ratio > threshold * 2. and len(measures) < len(r_) / 4:
+                    too_much_chaos = True
+                    break
+
+            if too_much_chaos is True:
```

Comment:
`if foobar is True` is an antipattern. The pythonic thing to do is `if foobar`.

Reply:
Thank for the advice.